[16.0][ADD] mgmtsystem_nonconformity_hazard: add new module (#576)
OCA/management-system: mgmtsystem_nonconformity_hazard/readme/DESCRIPTION.rst

> This is a bridge module between Management System and Management System

Reviewer: @kaynnan Can you be more specific?

Author: I made changes to the description, I believe it is now more detailed

> This is a bridge module between Management System and Hazard
> 
> This module contains some new features for Management System modules.
> 
> Nonconformity (NC)
> 
> - Add the hazard_ids field to link specific hazards.

Reviewer: ```suggestion
This module allows the user to relate hazards to a nonconformity.
```

Author: Adjustments completed, I only kept the new line mentioned
